src/app.js

```diff
@@ -0,0 +1,182 @@
+        <a href="#" type="button" class="task-option-remove"><i class="fa fa-trash"></i></a>
+      </div>
+      `;
+      div[i].innerHTML = template[i];
+      listBlock.appendChild(div[i]);
+    }
+  }
+
+  const update = () => {
+    const taskUpdate = document.querySelectorAll('.input-item');
+    taskUpdate.forEach((input, index) => {
+      input.addEventListener('click', () => {
+        const set = () => {
+          const updateTaskContent = JSON.parse(JSON.stringify(localStorage.getItem('taskList')));
+          const newUpdateTaskObj = JSON.parse(updateTaskContent);
+          newUpdateTaskObj[index].description = input.value;
+          localStorage.setItem('taskList', JSON.stringify(newUpdateTaskObj));
+          loadObj();
+        };
+        input.onchange = set;
+      });
+    });
+  };
+
+  const remove = (btn, key) => {
+    btn.addEventListener('click', () => {
+      const setTaskListR = JSON.parse(JSON.stringify(localStorage.getItem('taskList')));
+      const newTaskListObjR = JSON.parse(setTaskListR);
+
+      newTaskListObjR.splice(key, 1);
+
+      localStorage.setItem('taskList', JSON.stringify(newTaskListObjR));
+
+      resetObjIndex();
+      loadObj();
+    });
+  };
+
+  const taskOption = document.querySelectorAll('.task-option');
+  taskOption.forEach((button, index) => {
+    button.addEventListener('click', () => {
+      loadObj();
+
+      document.getElementById(`list-item-first_${index}`).style.display = 'none';
+      document.getElementById(`list-item-second_${index}`).style.display = 'flex';
+    });
+  });
+
+  const taskOptionRemove = document.querySelectorAll('.task-option-remove');
+  taskOptionRemove.forEach((button, index) => {
+    remove(button, index);
+  });
+
+  update();
+
+  const taskCheck = document.querySelectorAll('.task-check');
+  taskCheck.forEach((chk, index) => {
+    chk.addEventListener('click', () => {
+      if (chk.checked === true) {
+        document.getElementById(`label_${index}`).style.textDecoration = 'line-through';
+        document.getElementById(`label_${index}`).style.color = '#a9a9a9';
+
+        const chkTaskListSet = JSON.parse(JSON.stringify(localStorage.getItem('taskList')));
+        const newChkTaskListObjSet = JSON.parse(chkTaskListSet);
+        newChkTaskListObjSet[index].completed = true;
+        localStorage.setItem('taskList', JSON.stringify(newChkTaskListObjSet));
+      } else {
+        document.getElementById(`label_${index}`).style.textDecoration = 'none';
+        document.getElementById(`label_${index}`).style.color = '#000';
+
+        const chkTaskListSet = JSON.parse(JSON.stringify(localStorage.getItem('taskList')));
+        const newChkTaskListObjSet = JSON.parse(chkTaskListSet);
+        newChkTaskListObjSet[index].completed = true;
+        localStorage.setItem('taskList', JSON.stringify(newChkTaskListObjSet));
+      }
+    });
+  });
+
+  clearCompleted.addEventListener('click', () => {
```

Comment:
Hey @rolandfbk 
You've done a great job in your Todo List Project However there's an issue. When I delete an item from the to-do list. it doesn't update the indexes of the to-do list items.
#### For Example
- If I delete item # 2 from the to-do list then items # 3,4 should update their indexes to 2 and 3 respectively.
<img width="1440" alt="Screen Shot 2022-04-28 at 4 10 24 AM" src="https://user-images.githubusercontent.com/83048208/165645999-dcd56f93-03ac-490f-beba-df3bd9c13ce6.png">


- Currently, after selecting the todo item and when I click on `clear all completed` button. It's not updating the indexes. As you can see in the screenshot. Todo item # 3,4 should update their indexes to 2 and 3 after todo item # 2 gets deleted.
<img width="1435" alt="Screen Shot 2022-04-28 at 4 11 39 AM" src="https://user-images.githubusercontent.com/83048208/165646124-ddaa576f-bbbc-4f8d-a9e4-4f69cfe2be39.png">